internal/conf/parse_test.go

```diff
@@ -0,0 +1,94 @@
+package conf
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/sourcegraph/sourcegraph/internal/conf/conftypes"
+)
+
+func TestConfigChangeResult(t *testing.T) {
```

Comment:
```suggestion
func TestCalculateConfigChangeResult(t *testing.T) {
```

I think `calculateConfigChangeResult` is the function being tested.

Reply:
You are correct. Sloppy bit of late renaming on my part, sorry!